scripts/cray_runall.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+alps_present=`which alps`
+if [ $? == 0 ]; then
+launcher="aprun -n 1"
+else
+launcher="srun -n 1"
+fi
+echo Running fi_info
+$launcher ./fi_info 2>&1 | tee test.output
```

Comment:
if the build does a build/install of libfabric, followed by a build/install of fabtests, the script will end up in the same directory as the tests.  I was thinking about keeping things simple for jenkins regression tester.